### DISCUSSION TITLE
No Firebase App '[DEFAULT]' has been created - call firebase.initializeApp()

### DISCUSSION THREAD

**Discussion by badal8395** [invertase/react-native-firebase#4734] [Q&A] [answered]
hello,
i remove apply plugin: 'com.google.gms.google-services' in  /android/app/build.gradle file because i got
this error:
> Task :app:processDebugGoogleServices FAILED

so i removed 

so now i get this Error: No Firebase App '[DEFAULT]' has been created - call firebase.initializeApp()

please slove my problem

**Comment by mikehardy** [comment, marked as answer]
Please log an issue following details

Or read and run+explore https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh to see how to integrate the module
